languages/julia/concepts.csv

```diff
@@ -0,0 +1,74 @@
+concept,category
```

Comment:
It looks like most concepts are on a separate line, but here there are two on the same line. Is that correct?

Reply:
Yes, that's correct. those are the headers but in general, if there's only one entry in a line, it's just the concept. If there are two, it's the concept and it's category (like the directory names in the concepts folders).

Further down you can find some concept that are tagged with type-system.

Reply:
Okay cool! Just checking :)